org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/XMLAssert.java

```diff
@@ -1726,6 +1726,7 @@ public static void assertGrammarGenerator(String xml, FileContentGeneratorSettin
 
 	public static void assertGrammarGenerator(String xml, FileContentGeneratorSettings grammarSettings,
 			SharedSettings sharedSettings, String expected) {
+		sharedSettings.getFormattingSettings().setLegacy(true);

```

Comment:
We should look into rewritting the grammar generator tests to match the formatting of the new formatter. Also, I did some manual testing, and it looks like the test failures that happen when the experimental formatter is used for formatting generated grammars is legitimate (see https://github.com/eclipse/lemminx/issues/1382). I think we should address this before making the experimental formatter the default formatter.

Reply:
Once you rebase and fix the grammar generator tests, I think this will be ready to merge.

Reply:
sounds good! I'm also fixing some of the max line width tests since we decided to set the default value of max line width to 100 because of this issue: https://github.com/eclipse/lemminx/issues/1380